__tests__/Lotto.test.js

```diff
@@ -0,0 +1,131 @@
+import Lottos from "../src/domain/Lottos.js";
+import LottoPrizes from "../src/domain/LottoPrizes.js";
+import LottoGame from "../src/domain/LottoGame.js";
+import DrawNumbers from "../src/domain/DrawNumbers.js";
+
+describe("로또 게임", () => {
+  let lottoGame;
+
+  beforeEach(() => {
+    lottoGame = new LottoGame(new LottoPrizes());
+  });
+
+  it("로또는 1000원 단위로 구매할 수 있다. 구매 금액 1000원 당 로또 1장을 받을 수 있다.", () => {
+    const purchaseAmount = 10000;
+    const purchasedLottos = lottoGame.purchase(purchaseAmount);
+
+    const quantity = purchasedLottos.length;
+    expect(quantity).toBe(10);
+  });
+
+  it("로또 구입 시에는 1000으로 나누어 떨어지는 양수를 입력해야 한다.", () => {
```

Comment:
테스트 명만 보면 어떤 input에 대해 어떤 output을 기대하는지를 전혀 알 수 없는데요. 테스트명을 통해 이런 정보들을 드러내면 좋을거 같아요

```suggestion
  it("로또 구입 시에는 1000으로 나누어 떨어지지 않는 양수 4500을 입력받은 경우, '구입 금액은 1000원 단위로 입력해야 합니다.'. 메세지가 포함 된 Error를 Throw 해야 한다.", () => {
```